frontend/ui/data/koptoptions.lua

```diff
@@ -268,25 +285,45 @@ This can also be used to remove some gray background or to convert a grayscale o
                     return optionsutil.enableIfEquals(configurable, "text_wrap", 1)
                 end,
                 name_text_hold_callback = optionsutil.showValues,
+                help_text = _([[In reflow mode, sets the quality of the text and image extraction processing and output.]]),
             },
+        }
+    },
+    {
+        icon = "resources/icons/appbar.settings.large.png",
+        options = {
             {
-                name = "hw_dithering",
-                name_text = S.HW_DITHERING,
-                toggle = {S.ON, S.OFF},
-                values = {1, 0},
-                default_value = 0,
-                advanced = true,
-                show = Device:hasEinkScreen() and Device:canHWDither(),
+                name="doc_language",
+                name_text = S.DOC_LANG,
+                toggle = DKOPTREADER_CONFIG_DOC_LANGS_TEXT,
+                values = DKOPTREADER_CONFIG_DOC_LANGS_CODE,
+                default_value = DKOPTREADER_CONFIG_DOC_DEFAULT_LANG_CODE,
+                event = "DocLangUpdate",
+                args = DKOPTREADER_CONFIG_DOC_LANGS_CODE,
                 name_text_hold_callback = optionsutil.showValues,
+                help_text = _([[Set the language to be Used by the OCR engine.]]),
             },
             {
                 name = "forced_ocr",
                 name_text = S.FORCED_OCR,
-                toggle = {S.ON, S.OFF},
-                values = {1, 0},
+                toggle = {S.OFF, S.ON},
+                values = {0, 1},
                 default_value = 0,
                 advanced = true,
                 name_text_hold_callback = optionsutil.showValues,
+                help_text = _([[Force the use of OCR for text selection, even if the document has some text layer.]]),
```

Comment:
You might be talking about something technical with multiple text layers or whatnot? But I think that just unnecessarily complicates things.
```suggestion
                help_text = _([[Force the use of OCR for text selection, even if the document has a text layer.]]),
```